anaconda.spec.in

```diff
@@ -387,9 +386,11 @@ rm -rf \
 %{_bindir}/instperf
 %{_bindir}/anaconda-disable-nm-ibft-plugin
 %{_bindir}/anaconda-nm-disable-autocons
+%{_bindir}/run-in-new-session.py
```

Comment:
I think we should better put this helper script to `/usr/libexec/anaconda` - we already have a couple things there & the script has a rather specific use case. There quite a few ways how people can end up with Anaconda installed on their system, so better avoid putting unexpected stuff into their environment. :)

Reply:
Yes, please put helpers in libexec, also rename the file so it doesn't have `.py` suffix. That way the Python interpreter doesn't try to generate bytecode for it.

Reply:
Done. `/usr/libexec/anaconda` is not in `PATH` though, so I had to hardcode its path when running it. I'm not sure if this is something we need to/want to address.

Reply:
That is expected.